src/m365/spo/commands/navigation/navigation-node-add.spec.ts

```diff
@@ -189,6 +189,30 @@ describe(commands.NAVIGATION_NODE_ADD, () => {
     assert(loggerLogSpy.calledWith(nodeAddResponse));
   });
 
+  it('adds new navigation node to the top navigation with empty url', async () => {
+    const nodeAddResponse = {
+      "AudienceIds": audienceIds.split(','),
+      "CurrentLCID": 1033,
+      "Id": 2001,
+      "IsDocLib": true,
+      "IsExternal": false,
+      "IsVisible": true,
+      "ListTemplateType": 0,
+      "Title": "About",
+      "Url": "http://linkless.header/"
+    };
+    sinon.stub(request, 'post').callsFake(async (opts) => {
+      if (opts.url === `${webUrl}/_api/web/navigation/topnavigationbar`) {
+        return nodeAddResponse;
+      }
+
+      throw 'Invalid request';
+    });
+
+    await command.action(logger, { options: { webUrl: webUrl, location: 'TopNavigationBar', title: title, url: "", audienceIds: audienceIds, verbose: true } });
```

Comment:
Let's don't specify url at all. Also let's remove the audience IDs, we should test only the behavior of the linkless navigation labels here (as mentioned in the test title).